diff -rupN --no-dereference curl-7.73.0/configure curl-7.73.0-new/configure
--- curl-7.73.0/configure	2020-10-13 23:50:35.000000000 +0200
+++ curl-7.73.0-new/configure	2020-11-17 10:39:18.980130918 +0100
@@ -16354,18 +16354,11 @@ $as_echo "yes" >&6; }
     gccvhi=`echo $gccver | cut -d . -f1`
     gccvlo=`echo $gccver | cut -d . -f2`
     compiler_num=`(expr $gccvhi "*" 100 + $gccvlo) 2>/dev/null`
-    flags_dbg_all="-g -g0 -g1 -g2 -g3"
-    flags_dbg_all="$flags_dbg_all -ggdb"
-    flags_dbg_all="$flags_dbg_all -gstabs"
-    flags_dbg_all="$flags_dbg_all -gstabs+"
-    flags_dbg_all="$flags_dbg_all -gcoff"
-    flags_dbg_all="$flags_dbg_all -gxcoff"
-    flags_dbg_all="$flags_dbg_all -gdwarf-2"
-    flags_dbg_all="$flags_dbg_all -gvms"
+    flags_dbg_all=""
     flags_dbg_yes="-g"
     flags_dbg_off=""
-    flags_opt_all="-O -O0 -O1 -O2 -O3 -Os -Og -Ofast"
-    flags_opt_yes="-O2"
+    flags_opt_all=""
+    flags_opt_yes=""
     flags_opt_off="-O0"
 
     OLDCPPFLAGS=$CPPFLAGS
diff -rupN --no-dereference curl-7.73.0/m4/curl-compilers.m4 curl-7.73.0-new/m4/curl-compilers.m4
--- curl-7.73.0/m4/curl-compilers.m4	2020-10-09 08:57:07.000000000 +0200
+++ curl-7.73.0-new/m4/curl-compilers.m4	2020-11-17 10:39:18.981130916 +0100
@@ -175,18 +175,11 @@ AC_DEFUN([CURL_CHECK_COMPILER_GNU_C], [
     gccvhi=`echo $gccver | cut -d . -f1`
     gccvlo=`echo $gccver | cut -d . -f2`
     compiler_num=`(expr $gccvhi "*" 100 + $gccvlo) 2>/dev/null`
-    flags_dbg_all="-g -g0 -g1 -g2 -g3"
-    flags_dbg_all="$flags_dbg_all -ggdb"
-    flags_dbg_all="$flags_dbg_all -gstabs"
-    flags_dbg_all="$flags_dbg_all -gstabs+"
-    flags_dbg_all="$flags_dbg_all -gcoff"
-    flags_dbg_all="$flags_dbg_all -gxcoff"
-    flags_dbg_all="$flags_dbg_all -gdwarf-2"
-    flags_dbg_all="$flags_dbg_all -gvms"
+    flags_dbg_all=""
     flags_dbg_yes="-g"
     flags_dbg_off=""
-    flags_opt_all="-O -O0 -O1 -O2 -O3 -Os -Og -Ofast"
-    flags_opt_yes="-O2"
+    flags_opt_all=""
+    flags_opt_yes=""
     flags_opt_off="-O0"
     CURL_CHECK_DEF([_WIN32], [], [silent])
   else
